Modix.Data/Repositories/BehaviourConfigurationRepository.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System.Collections.Generic;
+using System.Threading.Tasks;
+using Microsoft.EntityFrameworkCore;
+using Modix.Data.Models;
+
+namespace Modix.Data.Repositories
+{
+    public interface IBehaviourConfigurationRepository
```

Comment:
Separate file, for consistency?

Reply:
I never quite understood why that's a thing - the interface has one concrete implementation, so it makes sense to keep it with that, no?

Reply:
Single entity per file is the convention, and some people like to follow it explicitly. I don't really mind either way, but we need to be consistent throughout the codebase.